src/main/java/frc/robot/autoCommands/autoDriveForward.java

```diff
@@ -57,7 +57,13 @@ public void execute() {
 
     currentPose = drivetrain.getRobotPose();
     velocity = accelConstant * (distance - currentPose.minus(initPose).getTranslation().getNorm());
+
+    if (velocity <= maxVelocity){
     drivetrain.setVelocity(velocity, velocity);
```

Comment:
could replace this using Java.lang.math.min()
drivetrain.setVelociey(min(velocity,maxVelocity),min(velocity,maxVelocity))